python/rikai/vision.py

```diff
@@ -167,3 +171,49 @@ def to_numpy(self) -> np.ndarray:
             "bbox": self.bbox.to_numpy(),
             "score": self.score,
         }
+
+
+class VideoStream:
```

Comment:
Move these two classes to "rikai.types.vision" or `rikai.types.video`?

Reply:
ok i can rebase on your changes